### PR TITLE
Create harmeet_singh

### PR DESCRIPTION
**Pull request by harmeet24**
I would like to register for JS CraftCamp.

- [x] My pull request contains a JSON file `$name_or_nickname.json`
- [x] I read the `THE IMPORTANT STUFF` at [https://jscraftcamp.org/registration](https://jscraftcamp.org/registration), the JSON file follows the [template](https://github.com/jscraftcamp/website/blob/main/participants/_template.json), and contains the mandatory fields like `realName`, `githubAccountName`, `when`, `iCanTakeNotesDuringSessions`, `whatIsMyConnectionToJavascript` and `whatCanIContribute`.
- [x] If I can NOT attend, I will either send another pull request removing my JSON file or an e-mail with the subject 'UNREGISTER' to team@jscraftcamp.org
- [x] I agree that the data I enter in the registration can be used for running the event, e.g. make a name tag, count me as participant. I acknowledge that my data are public on GitHub and I agree that I will be listed on the JSCraftCamp website as a participant.
- [x] I agree that photos and videos might be taken and published (e.g. on social media) during the event.
- [x] I understand that I might NOT get a T-Shirt, because they might be in production already
- [ ] I asked my company about sponsoring (see open items at: https://github.com/orgs/jscraftcamp/projects/7)

Are you from a sponsoring company?

- [ ] Yes, I am from company ?????

### CODE REVIEW

**Review by meldron — CHANGES_REQUESTED** [review]
Hey @harmeet24, please add the json file extension to your newly added file!

File: participants/harmeet_singh.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	"whatIsMyConnectionToJavascript": "From the beginning of my career, I never get a chance to work on JavaScript and explore JavaScript-based technologies.",
+	// what can you contribute to the bar camp (required)
+	"whatCanIContribute": "I am happy to discuss backend technologies like Java and Scala and happy to explore distributed systems architecture."
+	// if you want a T-Shirt we need your size and variant preference (optional)
+	// the following sizes are available: M (only regular cut)
+	"tShirtSize": "M",
+	// your LinkedIn profile URL
+	"linkedin": "https://www.linkedin.com/in/harmeet-singh-taara-1654165b/",
+	// your X (Twitter) handle - must match regex ^[a-zA-Z_]{1}[a-zA-Z0-9_]{0,14}$ (optional)
+	"X": "@singh_harmeet13",
```
Comment (Narigo) [inline participants/harmeet_singh.json:47]: I think the `@` sign needs to be removed here

```suggestion
	"X": "singh_harmeet13",
```

**Review by Narigo — COMMENTED** [review]
Hi @harmeet24 sorry, there is another change to be made. Looks like the `X` value didn't go through - no need for the `@` sign there!

**Comment by harmeet24** [comment]
> Hi @harmeet24 sorry, there is another change to be made. Looks like the `X` value didn't go through - no need for the `@` sign there!

I fixed the issue, please have a look :)

**Comment by harmeet24** [comment]
> Hey @harmeet24, please add the json file extension to your newly added file!

done

**Review by meldron — APPROVED** [review]

**Comment by meldron** [comment]
Thanks a lot for fixing the PR, welcome to JSCC, see you there!

**Pull request merged**